src/meson.build

```diff
@@ -532,7 +532,7 @@ if have_icu and not have_icu_builtin
 
   pkgmod.generate(libharfbuzz_icu,
     description: 'HarfBuzz text shaping library ICU integration',
-    requires: ['harfbuzz'],
+    requires: ['harfbuzz = @0@'.format(meson.project_version())],
```

Comment:
Just for educational reasons, is there any reason that `icuuc` isn't here but `glib-2.0` is put on gobject, icu's versioning and so?

Reply:
I'm not sure. The template files for autotools have `icuuc` only as a private dependency. The generator already does this so there was no need to list it.

https://github.com/harfbuzz/harfbuzz/blob/22ce0961742df0a213f2cc86ca3d5a72453c00b1/src/harfbuzz-icu.pc.in#L10-L11

Reply:
Ok so, thanks :)